README.md

```diff
@@ -36,6 +37,9 @@ The fields have to be one of the types that the sync package supports in order t
 - sync.Int64, allows for concurrent int64 manipulation
 - sync.Float64, allows for concurrent float64 manipulation
 - sync.Bool, allows for concurrent bool manipulation
+- sync.Secret, allows for concurrent secret manipulation. Secrets can only be strings
+
+For sensitive configuration (passwords, tokens etc.) that shouldn't be printed in log, you can use the `Secret` flavor of `sync` types. If one of these is selected, then at harvester log instead of the real value the text `***` will be displayed.
```

Comment:
Missing a comma after `(passwords, tokens`.

Reply:
Well. To be honest I thought that this was the right way to write a list of elements and skip the last comma, but it seems not only should be used, but it also has a name ([Serial Comma](https://en.wikipedia.org/wiki/Serial_comma)) :smile: